c++.html.markdown

```diff
@@ -867,6 +868,99 @@ v.push_back(Foo());  // New value is copied into the first Foo we inserted
 // explanation of why this works.
 v.swap(vector<Foo>());
 
+
+/////////////////////
+// CONTAINERS
+/////////////////////
+
+//Containers or the Standard Template Library are some predefined templates
```

Comment:
please add a space between the `//` and the text, like the rest of the tutorial does